stable/rabbitmq/values-production.yaml

```diff
@@ -293,6 +293,25 @@ metrics:
     ##
     # pullSecrets:
     #   - myRegistryKeySecretName
+    
+  ## Prometheus Service Monitor
+  ## ref: https://github.com/coreos/prometheus-operator
+  ##      https://github.com/coreos/prometheus-operator/blob/master/Documentation/api.md
+  serviceMonitor:
+    ## If the operator is installed in your cluster, set to true to create a Service Monitor Entry
+    enabled: true
```

Comment:
Do not enable it by default in the **values-production.yaml**.
ServiceMonitor is a CRD and it won't be available in the API unless the Prometheus Operator is installed in the cluster. Therefore, the user must implicitly choose to enable it.

Reply:
It seems to me that, if a CRD is not available in the API, helm not returns errors; it simply not create the resource.
However, you are right and I followed your advice. Now the user must explicitly choose to enable it.